pinot-core/src/test/java/org/apache/pinot/core/query/selection/SelectionOperatorServiceTest.java

```diff
@@ -235,11 +238,11 @@ public void testCompatibleRowsRenderSelectionResultsWithoutOrdering() {
     List<Serializable[]> resultRows = selectionResults.getRows();
     Serializable[] expectedRow1 = {
         0L, 1.0, 2.0, 3.0, "4", new long[]{5L}, new double[]{6.0}, new double[]{7.0}, new double[]{8.0},
-        new String[]{"9"}, "1020"
+        new String[]{"9"}, "1020", "2022-05-02 14:47:35.0", new String[]{"2022-05-02 14:47:35.0"}
```

Comment:
new tests asserting timestamp and timestamp array formatting will only run on machines with date/time set to UTC

Reply:
You may create the expected result using `Timestamp` class instead of hard-coded strings